ui/src/app/base/sagas/websockets.ts

```diff
@@ -545,7 +558,8 @@ const buildMessage = (
     method: `${meta.model}.${meta.method}`,
     type: WebSocketMessageType.REQUEST,
   };
-  if (params && !Array.isArray(params)) {
+  const hasMultipleDispatches = meta.dispatchMultiple && Array.isArray(params);
```

Comment:
Previously we didn't have any websocket endpoints that took an array of params so we used an array to make the websocket handler send multiple events. Now I've made multiple dispatches explicit with `dispatchMultiple`.